### ISSUE TITLE
Quicker setRecord

### ISSUE THREAD

**Issue by Joeasaurus** [chilts/nice-route53#11]
It'd be nice if I could choose to update a record instead of deleting and creating all the time.

**Comment by chilts** [comment]
Hi @Joeasaurus, thanks for raising this issue.

I can't remember why, but I think there was a reason I did it as a delete and a create, however, that reason is lost in the annals of time and history.

I don't quite have the time to do this at the moment, but I'd be happy to accept a PR - it shouldn't be too hard I don't think. Or maybe @leorossi might be able to take a look.

**Comment by leorossi** [comment]
I have taken a look and my guess is that if you change the `Action` parameter here https://github.com/chilts/nice-route53/blob/master/nice-route53.js#L381 from `CREATE` to `UPSERT` you should keep the old behavior, plus have the update feature you are requesting.

If you can try and test it, it will be the quickest PR ever :)

**Comment by Joeasaurus** [comment]
@leorossi I'm of the same mind, but won't it then still push 1 x DELETE and 1 x UPSERT? Surely that's no better than delete + create?

**Comment by leorossi** [comment]
Oh you are right, I think that commenting this part https://github.com/chilts/nice-route53/blob/master/nice-route53.js#L361-L377 and changing the following `Action` to `UPSERT` will make the job quicker, right?

**Comment by Joeasaurus** [comment]
Yeah that's more like what I was thinking, or a boolean in `opts` to choose update or delete + create.

**Comment by c0bra** [comment]
Any movement on this? Was looking to perform UPSERT operations,.

**Comment by rashkov** [comment]
upsert is now available, it was merged here https://github.com/chilts/nice-route53/pull/15
